scripts/mockdata/objects_e1s4.json

```diff
@@ -0,0 +1,225 @@
+      "blackboardData": {
+        "componentType": "rotation-controls",
+        "jsonData": {
+          "position": [0, 0, 5],
+          "scale": [0.8, 0.8, 0.8]
+        }
+      }
+    }
+  },
+  {
+    "name": "Phone",
+    "position": [3.941, 2.166, -3.48],
+    "scale": [0.3, 0.3, 0.3],
+    "rotation": [0, 280, 0],
+    "asset_id": 36,
+    "next_objects": [],
+    "texts": [],
+    "is_interactable": true,
+    "animations_json": {
+      "blackboardData": {
+        "componentType": "visual-pane",
+        "jsonData": {
+          "imageSrc": "https://dcc-bp-public-dev.s3.us-east-2.amazonaws.com/images/62521e3397d6a781018182426caf6b6c.png",
+          "caption": ""
+        }
+      }
+    }
+  },
+  {
+    "name": "Friends",
+    "position": [-1.855, 3.667, -5.443],
+    "scale": [0.073, 0.073, 0.073],
+    "rotation": [0, 270, 0],
+    "asset_id": 69,
+    "next_objects": [],
+    "texts": [],
+    "is_interactable": true,
+    "animations_json": {
+      "blackboardData": {
+        "componentType": "visual-pane",
+        "jsonData": {
+          "imageSrc": "https://dcc-bp-public-dev.s3.us-east-2.amazonaws.com/images/44a5ebe14f3ed2b854ae0ab5f2f3e438.jpeg",
+          "caption": ""
+        }
+      }
+    }
+  },
+  {
+    "name": "Concert",
+    "position": [-3.283, 3.809, -5.443],
+    "scale": [0.073, 0.073, 0.073],
+    "rotation": [0, 270, 0],
+    "asset_id": 70,
+    "next_objects": [],
+    "texts": [],
+    "is_interactable": true,
+    "animations_json": {
+      "blackboardData": {
+        "componentType": "visual-pane",
+        "jsonData": {
+          "imageSrc": "https://dcc-bp-public-dev.s3.us-east-2.amazonaws.com/images/d9fee8c2e9ab1bd457232bfcae704dd7.jpeg",
+          "caption": ""
+        }
+      }
+    }
+  },
+  {
+    "name": "Kalimba",
+    "position": [2.286, 3.093, 2.711],
+    "scale": [0.9, 0.9, 0.9],
+    "rotation": [0, 19.6, 0],
+    "asset_id": 71,
+    "next_objects": [],
+    "texts": [],
+    "is_interactable": true,
+    "animations_json": {
+      "blackboardData": {
+        "blackboardText": "Kalimba",
+        "blackboardParagraph": "I really love the sound of my kalimba. It's really gentle sounding, especially when you play each key one by one starting from the highest note then working down. My hands are still so shaky I wouldn't be able to play properly. I also don't want my parents to know that I am home so I can't make any sounds.",
+        "componentType": "rotation-controls",
+        "jsonData": {
+          "position": [0, 0, 5],
+          "scale": [1.5, 1.5, 1.5]
+        }
+      }
+    }
+  },
+  {
+    "name": "Mirror",
+    "position": [-4.291, 0.022, 2.872],
+    "scale": [0.71, 1, 1],
+    "rotation": [-12.911, -179.13, 0.9],
+    "asset_id": 72,
+    "next_objects": [],
+    "texts": [],
+    "is_interactable": true,
+    "animations_json": {
+      "blackboardData": {
+        "componentType": "keypad",
+        "blackboardText": "kcab ,daeh ,sdanh ,teef \\\\ 5->",
```

Comment:
I am a bit confused about this puzzle lol, but I guess it's a WIP based on what DCC tells us